<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-10</y>
     <width>1381</width>
     <height>751</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Downloads/bg_thing_edited4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>152</y>
     <width>551</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Arial Black', 'Impact', sans-serif;
    font-size: 40px;
    font-weight: bold;
    color: #4B2E1E; 
    letter-spacing: 0.5px;</string>
   </property>
   <property name="text">
    <string>Welcome Back!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFC107;  /* Bright yellow */
    color: #000000;             /* Black text */
    font-weight: bold;
    font-size: 14px;
    border: none;
    border-radius: 8px;         /* Rounded corners */
    padding: 10px 0;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>122</y>
     <width>491</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 250, 199);
    border-radius: 15px;
    padding: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>252</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
   </property>
   <property name="text">
    <string>Email andress</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>352</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>212</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
   </property>
   <property name="text">
    <string>Please login to your account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_email">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>282</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e0e0e0; 
    border-radius: 10px;    
    padding: 8px 12px;                
    background-color: #ffffff;        
    color: #333333;                    
    font-size: 14px; </string>
   </property>
   <property name="placeholderText">
    <string>Input your email here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_password">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>382</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e0e0e0; 
    border-radius: 10px;    
    padding: 8px 12px;                
    background-color: #ffffff;        
    color: #333333;                    
    font-size: 14px; </string>
   </property>
   <property name="placeholderText">
    <string>Input your password here</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>450</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
   </property>
   <property name="text">
    <string>Forgot password?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>530</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FFC107;  /* Bright yellow */
    color: #000000;             /* Black text */
    font-weight: bold;
    font-size: 14px;
    border: none;
    border-radius: 5px;         /* Rounded corners */
    padding: 7px 0;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eye">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/eye-slash-solid.svg</normaloff>../img/eye-slash-solid.svg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Lucida Sans Unicode&quot;;</string>
   </property>
   <property name="text">
    <string>Don't have an account yet?</string>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>label</zorder>
  <zorder>btn_login</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>txt_email</zorder>
  <zorder>txt_password</zorder>
  <zorder>label_6</zorder>
  <zorder>btn_register</zorder>
  <zorder>btn_eye</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
